src/main/java/org/traccar/api/ResourceErrorHandler.java

```diff
@@ -15,20 +15,33 @@
  */
 package org.traccar.api;
 
+import org.traccar.config.Config;
+import org.traccar.config.Keys;
 import org.traccar.helper.Log;
 
+import jakarta.inject.Inject;
 import jakarta.ws.rs.WebApplicationException;
 import jakarta.ws.rs.core.Response;
 import jakarta.ws.rs.ext.ExceptionMapper;
 
 public class ResourceErrorHandler implements ExceptionMapper<Exception> {
 
+    private static final String HIDDEN_MESSAGE_KEY = "HiddenError";
```

Comment:
This doesn't make sense to me. I think we should return at least the message.

Reply:
How would you do it without revealing technical message ?

Reply:
You don't. The message is needed and intended for the end user in most cases.